ioctl/newcmd/bc/bcblock.go

```diff
@@ -144,14 +139,6 @@ type actionInfo struct {
 	Signature    []byte `protobuf:"bytes,3,opt,name=signature,proto3" json:"signature,omitempty"`
 }
 
-func (m *blockMessage) String() string {
-	if output.Format == "" {
-		message := fmt.Sprintf("Blockchain Node: %s\n%s\n%s", m.Node, output.JSONString(m.Block), output.JSONString(m.Actions))
-		return message
-	}
-	return output.FormatString(output.Result, m)
-}
-
 // getActionInfoByBlock gets action info by block hash with start index and action count
 func getActionInfoWithinBlock(cli *iotexapi.APIServiceClient, height uint64, count uint64) ([]*iotexapi.BlockInfo, error) {
```

Comment:
can combine getBlockMetaByHeight  and getActionInfoWithinBlock ?  because two function are similiar